Models/RomM/Rom/RomMSimilarGame.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+
+namespace RomM.Models.RomM.Rom
+{
+    public class RomMSimilarGame
+    {
+        [JsonProperty("id")]
+        public int Id { get; set; }
+
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        [JsonProperty("slug")]
+        public string Slug { get; set; }
+
+        [JsonProperty("type")]
+        public string Type { get; set; }
+
+        [JsonProperty("cover_url")]
+        public string CoverUrl { get; set; }
+    }
+}
```

Comment:
these models are nice! are they like type definitions, in that they're used for intellisense?

i wonder if there's a way to generate them from the existing schema, maybe via the openapi spec...

Reply:
Yes, i basically replaced your dynamic access with fully typed objects which also doesn't require reflection and is way faster because of that

Technically we could try to use generation but i usually just grab the json from a call and throw it in a converter which then produces those types for me (like i did here as well 😅)